actors/builtin/codes.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 // The built-in actor code IDs
-var SystemActorCodeID cid.Cid
+var SystemActorCodeID cid.Cid // todo: drop id?
```

Comment:
Let's leave it for now. It's definitely not the code itself, but an identifier of the code.